app/controllers/admin/users_controller.rb

```diff
@@ -35,6 +35,9 @@ def edit
     def show
       @user = User.find(params[:id])
 
+      set_feedback_messages
+      set_related_reactions
```

Comment:
This is likely very wrong but I needed to move it here just to be display this stuff on /show.html.erb view.